src/ServiceProvider.php

```diff
@@ -130,4 +130,10 @@ protected function getAppConfig(): array
 
         return $config;
     }
+
+    private function isAgentDisabled(): bool
+    {
+        return false === config('elastic-apm-laravel.active')
```

Comment:
@arkaitzgarro @countless-integers This is duplicated logic with the ApmCollectorService, and the job middleware could safely share it as well. Any advice on making it more available? It could go in the service for access via facade, but I'm not sure if something else makes more sense.
What makes me hesitate on that idea is that the facade isn't going to load until it's needed, but if we call it from the ServiceProvider it will always load.

Reply:
I have no problem with just checking the config value tbh. It's usually the first thing we want to check before loading anything. Could have a facade method, but then it would mean two ways of checking the same thing.

Reply:
I'm also ok for now. We could extend [philkra's config class](https://github.com/philkra/elastic-apm-php-agent/blob/master/src/Helper/Config.php) with some handy methods, but definetely not in this PR.